tools/install.py

```diff
@@ -92,3 +93,9 @@ def run(ip_addr, hostname, config, scripts, user, toolchain, **kwargs):
 
     if config in ['ignore', 'i']:
         cprint('Ignoring configuration changes', 'blue', attrs=['bold'])
+        
+    # Pipe the git commit to file
+    versionFile = open("version.txt", "w")
+    subprocess.Popen("git log -1", stdout=versionFile, shell=True)
```

Comment:
Don't use shell=true unless you really really need to. It can represent a security risk. Instead the first argument to this function should be an array of strings.